### DISCUSSION TITLE
Permission errors on latest Windows 11 update

### DISCUSSION THREAD

**Discussion by raylaxing** [LizardByte/community#465] [Sunshine]
Hi,
I've been using sunshine great for a year or so. Yesterday I did the 2024-06 Cumulative update for Win11-23h2-x64 and now I can't appear to connect. My shortcuts in sunshine are for setting screen resolution, so i can dynamically set it correctly per device, but i get this error:

```
[2024:06:12:11:42:28]: Info: Executing: [powershell C:\Users\user\720.ps1] in ["C:\Windows\System32\WindowsPowerShell\v1.0"]
[2024:06:12:11:42:28]: Warning: Couldn't run [powershell C:\Users\user\720.ps1]: System: Permission denied
```

Just to confirm, this script path and script has not been touched. SYSTEM has full r/w access to the whole C:/ drive too, I've double checked this. I have tried toggling 'run as admin' on the shortcuts in sunshine and still the same issue arises. Running the ps1 script in rdp (while pointless) does not error and returns a successful message - there's nothing in the script breaking the functionality. This happened only after the last system update last night, and that's the only thing that's changed.
- also i have tried re-installing sunshine and i get the same result. The tray icon no longer appears in the system tray but I can still access the web browser interface just fine. There's no errors I can see in the system event viewer

Any ideas? Thanks

**Comment by raylaxing** [comment]
Just confirmed, rolled back to before the update with a system restore and it's all working perfectly fine again

**Reply by Kenny417** [reply]
I ran into this problem because I was connected to the gaming PC via Remote Desktop (RDP).  Once I physically logged into the computer, Moonlight started working again.

**Comment by ReenigneArcher** [comment]
Seems like you might be running a less privileged instance of Sunshine...

**Reply by raylaxing** [reply]
I would normally agree with you - aside from the fact rolling the new Windows 11 system update back to the previous version fixed the issue.

**Comment by cchance27** [comment]
Same issue installed a windows cumulative update and Poof permissions errors, trying to rollback now

**Reply by pmarsh** [reply]
Did the rollback work?

Unfortunately just installed Sunshine for the first time on 2024-07 cumulative update Windows 11 Version 23H2 for x64-based and getting similar errors even when config is set to run as Admin.

[2024:07:30:17:19:14]: Info: Executing: [steam://open/bigpicture] in [""]
[2024:07:30:17:19:14]: Warning: Couldn't run [steam://open/bigpicture]: System: Permission denied

**Reply by Baldur404** [reply]
same issue,i was used admin Permission run "Sunshine" application ,I'm windouws 11 OS too

**Comment by albeec13** [comment]
Installed Sunshine for the first time yesterday and running into this issue as well on Windows 11.

```
[2025-06-30 16:34:46.234]: Info: Executing: [steam://open/bigpicture] in [""]
[2025-06-30 16:34:46.234]: Warning: Couldn't run [steam://open/bigpicture]: System: Permission denied
```

Up to date Windows 11 system running as a VM on QEMU for what it's worth. Headless, with RDP used to connect.

I was able to get to a desktop successfully, but I'm trying to connect from a handheld gaming device that does not support keyboard or mouse, and I am unable to interact with the desktop login page at all. I can just see that it loads. Steam gives me the error above, as does anything else I tried to add, despite the Local System user having permissions to run everything.

[a comment by an automated account is not shown]
